main.js

```diff
@@ -63,8 +67,7 @@ const createRecord = () => {
     process.exit(1);
   }
 
-  console.log(`echo "id=${result.id}" >> $GITHUB_ENV`);
-  console.log(`echo "name=${result.name}" >> $GITHUB_ENV`);
```

Comment:
I noticed that these were redirecting to `$GITHUB_ENV`, but seeing that these evolved from `::set-output`, I think these too should be redirecting to `$GITHUB_OUTPUT` per https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/